jax_distribution/requirements.txt

```diff
@@ -7,4 +7,8 @@ optax
 flax
 numpy
 brax
+sacred
+tensorboard_logger
+hydra-core==1.3.2
+omegaconf
```

Comment:
I think `omegaconf` comes with Hydra by default. 
Something else we'd have to set now as well is protobuf. Our code won't run if we have `protobuf > 3.20.x`. Could you try running the code in a clean Conda env to just check that our requirements file works? 
We can also remove `brax`.

Reply:
I ended up just making a slimmer requirements file.